cmake/dependencies.cmake

```diff
@@ -56,38 +55,3 @@ CPMAddPackage(
   GITHUB_REPOSITORY boost-ext/reflect
   GIT_TAG v1.1.1
 )
-
-############################################################################################################################
-# Packages needed for tt-metal simulator
-#   NNG for IPC/TCP communication
-#   Google Flatbuffers for serialization
-#   libuv for process mgmt
-############################################################################################################################
-CPMAddPackage(
```

Comment:
Ah because they're only in UMD right?

Reply:
yep only used in umd so moved them umd, metal will still bring in the packages regardless but now it's more organized